tests/Unit/Http/ResponseTest.php

```diff
@@ -129,6 +129,7 @@ public function testCors()
             $this->assertContains('Access-Control-Allow-Origin: *', $headers);
             $this->assertContains('Access-Control-Allow-Headers: Content-Type', $headers);
             $this->assertContains('Access-Control-Allow-Methods: GET, POST, PUT, DELETE', $headers);
+            $this->assertContains('Access-Control-Allow-Credentials', $headers);
```

Comment:
I think you forgot to assert the header value (`true`) besides its presence.
```suggestion
            $this->assertContains('Access-Control-Allow-Credentials: true', $headers);
```

Reply:
Thank you!  I see it now, lets see how THAT goes!